content/livesync/models/models.textile

```diff
@@ -93,30 +92,26 @@ SELECT COALESCE(MAX(sequence_id), 0) FROM outbox;
 
 h3(#history). Replay from history
 
-After the Models SDK initializes the model state via the sync function, it is important to apply the exact sequence of change events from your backend from the correct point in the stream. This position is specified by the @sequenceID@ returned from your backend endpoint.
+The SDK must initialize the model's state using the sync function and then precisely apply a sequence of change events from your backend, beginning at the correct point in the stream. This starting point is the @sequenceID@ your backend endpoint returns.
 
-Internally, after calling the sync function, the SDK attaches to the "channel":/channels and "subscribes":/channels to live messages. Live messages are buffered and not yet processed. It then paginates backwards through the message history on the channel from the point of attachment using "@untilAttach@":/storage-history/history#until-attach. The number of items in each page can be configured via the @syncOptions.historyPageSize@ defined in your "@ClientOptions@":/livesync/models#options.
+After the SDK executes the sync function, it internally connects to the "channel":/channels and begins "subscribing":/channels to live, yet unprocessed, messages.  It then paginates backward through the channel's message history, starting from the attachment point, using "@untilAttach@":/storage-history/history#until-attach. The number of items on each page can be configured via the @syncOptions.historyPageSize@ defined in your "@ClientOptions@":/livesync/models#options. 
 
-When a message with the target @sequenceID@ is reached, all of the messages from that point forward are sent to the model for processing to maintain continuity with incoming live messages.
+When a message is sent to the @sequenceID@, the model processes subsequent messages to maintain continuity with live messages. If the target @sequenceID@ is unreachable, there is insufficient message history to resume from the correct point to update the model. The SDK will attempt a re-sync. It re-syncs by calling the sync function again to acquire a newer version of the model state.
 
-If the target @sequenceID@ cannot be reached, there is insufficient message history to resume from the correct point and bring the model up to date. The SDK will try to sync again with a fresher version of the model state by calling your sync function once again.
+The amount of history available to query on the channel is determined by your "message storage":/storage-history/storage configuration on the channel. This configuration must match the @syncOptions.messageRetentionPeriod@ defined in your "@ClientOptions@":/livesync/models#options. The SDK uses this configuration option as a hint as to whether to resynchronize via the sync function and skip paginating through history when messages are expected to have expired.
```

Comment:
> and skip paginating through history when messages are expected to have expired

This isn't correct, when the sync function is called again it will still need to look through history